gr-blocks/lib/udp_sink_impl.cc

```diff
@@ -101,13 +101,16 @@ int udp_sink_impl::work(int noutput_items,
                         gr_vector_void_star& output_items)
 {
     const char* in = (const char*)input_items[0];
-    ssize_t r = 0, bytes_sent = 0, bytes_to_send = 0;
-    ssize_t total_size = noutput_items * d_itemsize;
+    size_t r = 0;
+    std::ptrdiff_t bytes_sent = 0, bytes_to_send = 0;
+    size_t total_size = noutput_items * d_itemsize;
```

Comment:
```suggestion
    const size_t total_size = noutput_items * d_itemsize;
```
...but I'm not a stickler for it.